fabric-loot-table-api-v2/src/main/java/net/fabricmc/fabric/api/loot/v2/FabricLootPoolBuilder.java

```diff
@@ -0,0 +1,181 @@
+		return this;
+	}
+
+	@Override
+	public FabricLootPoolBuilder with(LootPoolEntry.Builder<?> entry) {
+		super.with(entry);
+		return this;
+	}
+
+	@Override
+	public FabricLootPoolBuilder conditionally(LootCondition.Builder condition) {
+		super.conditionally(condition);
+		return this;
+	}
+
+	@Override
+	public FabricLootPoolBuilder apply(LootFunction.Builder function) {
+		super.apply(function);
+		return this;
+	}
+
+	// Custom methods
+
+	/**
+	 * Adds an entry to this builder.
+	 *
+	 * @param entry the added loot entry
+	 * @return this builder
+	 */
+	public FabricLootPoolBuilder with(LootPoolEntry entry) {
+		access.getEntries().add(entry);
+		return this;
+	}
+
+	/**
+	 * Adds entries to this builder.
+	 *
+	 * @param entries the added loot entries
+	 * @return this builder
+	 */
+	public FabricLootPoolBuilder with(Collection<? extends LootPoolEntry> entries) {
+		access.getEntries().addAll(entries);
+		return this;
+	}
+
+	/**
+	 * Adds a condition to this builder.
+	 *
+	 * @param condition the added condition
+	 * @return this builder
+	 */
+	public FabricLootPoolBuilder conditionally(LootCondition condition) {
+		access.getConditions().add(condition);
+		return this;
+	}
+
+	/**
+	 * Adds conditions to this builder.
+	 *
+	 * @param conditions the added conditions
+	 * @return this builder
+	 */
+	public FabricLootPoolBuilder conditionally(Collection<? extends LootCondition> conditions) {
+		access.getConditions().addAll(conditions);
+		return this;
+	}
+
+	/**
+	 * Applies a function to this builder.
+	 *
+	 * @param function the applied loot function
+	 * @return this builder
+	 */
+	public FabricLootPoolBuilder apply(LootFunction function) {
+		access.getFunctions().add(function);
+		return this;
+	}
+
+	/**
+	 * Applies loot functions to this builder.
+	 *
+	 * @param functions the applied loot functions
+	 * @return this builder
+	 */
+	public FabricLootPoolBuilder apply(Collection<? extends LootFunction> functions) {
+		access.getFunctions().addAll(functions);
+		return this;
+	}
+
+	/**
+	 * Copies the entries, conditions and functions of the {@code pool} to this
+	 * builder.
+	 *
+	 * <p>This is equal to {@code copyFrom(pool, false)}.
+	 */
+	public FabricLootPoolBuilder copyFrom(LootPool pool) {
+		return copyFrom(pool, false);
+	}
+
+	/**
+	 * Copies the entries, conditions and functions of the pool to this builder.
+	 *
+	 * <p>If {@code copyRolls} is true, the rolls of the pool are also copied.
+	 */
+	public FabricLootPoolBuilder copyFrom(LootPool pool, boolean copyRolls) {
+		access.getConditions().addAll(FabricLootPools.getConditions(pool));
+		access.getFunctions().addAll(FabricLootPools.getFunctions(pool));
+		access.getEntries().addAll(FabricLootPools.getEntries(pool));
+
+		if (copyRolls) {
+			rolls(FabricLootPools.getRolls(pool));
+		}
+
+		return this;
+	}
+
+	/**
+	 * Creates an empty loot pool builder.
+	 *
+	 * @return the created builder
+	 */
+	public static FabricLootPoolBuilder builder() {
```

Comment:
```java
FabricLootPoolBuilder.builder()
```
looks a bit weird in usage.

Reply:
I think in this case since the builder is being made it would become `FabricLootPoolBuilder.create()`

Reply:
I'll change these to `create`.